src/SSW.Rewards.Application/Quizzes/Commands/AddNewQuiz/AddNewQuizCommand.cs

```diff
@@ -0,0 +1,85 @@
+﻿using SSW.Rewards.Application.Achievements.Common;
+using SSW.Rewards.Application.Quizzes.Common;
+using SSW.Rewards.Domain.Enums;
+
+namespace SSW.Rewards.Application.Quizzes.Commands.AddNewQuiz;
+public class AddNewQuizCommand : IRequest<int>
+{
+    public QuizDetailsDto Quiz { get; set; }
```

Comment:
Don't reuse this object - create a new one that excludes things like QuizId, etc.

Reply:
Done